src/components/admin/ChangesViewer/ChangesSidebar.tsx

```
@@ -0,0 +1,74 @@
+                <ul className="space-y-0.5 px-2">
+                    {availablePages.map((page) => (
+                        <li key={page.id}>
+                            <button
+                                onClick={() => onPageSelect(page.id)}
+                                className={cn(
+                                    "w-full flex items-center gap-2 px-2 py-1.5 text-sm rounded-md transition-colors",
+                                    selectedPage === page.id
+                                        ? "bg-sidebar-accent text-sidebar-accent-foreground font-medium"
+                                        : "text-sidebar-foreground hover:bg-sidebar-accent/50 hover:text-sidebar-accent-foreground"
+                                )}
+                            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)